src/hooks/useEngineConnectionSubscriptions.ts

```diff
@@ -56,6 +56,7 @@ export function useEngineConnectionSubscriptions() {
         ;(async () => {
           if (stateRef.current.matches('Sketch no face')) return
           const event = await getEventForSelectWithPoint(engineEvent)
+          console.log('event', event)
```

Comment:
This is useful for those less familiar with the artifact Graph to see what the underlying selections is because it helps write the source range context.